src/dashboard/src/page_manager.c

```diff
@@ -430,6 +434,34 @@ void draw_volt_lim_page(page_manager_t *pm, NHD_US2066_OLED *oled) {
     oled_print(oled, ">");
 }
 
+void draw_launch_control_page(page_manager_t *pm, NHD_US2066_OLED *oled) {
+    carstats_t *stats = pm->stats;
+
+    oled_clearline(oled, 1);
+    oled_set_pos(oled, 1, 0);
+
+    if (stats->buttons.right.rising_edge) {
+        stats->controls.launch_ctrl_slip_ratio += 1;
+    }
+
+    stats->controls.launch_ctrl_slip_ratio = LOOPOVER(stats->controls.launch_ctrl_slip_ratio, 101, 120);
```

Comment:
What does LOOPOVER do? Is it a custom construct?

Reply:
It is. If the value goes above the third argument then it loops over to the second argument using modulo.